src/Microsoft.Data/Microsoft.Data.DataFrame.csproj

```diff
@@ -18,6 +18,11 @@
     <PackageReference Include="System.Memory" Version="4.5.1" />
     <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.1" />
     <PackageReference Include="System.Text.Encoding" Version="4.3.0" />
+    <PackageReference Include="Microsoft.ML" Version="1.1.0" />
```

Comment:
Don't reference the whole `Microsoft.ML`.  But instead, `Microsoft.ML.DataView`.

I wonder if that functionality should be split into a separate assembly. That way the core DataFrame library doesn't need to depend on Microsoft.ML.